src/features/Sync/SyncSlice.ts

```diff
@@ -22,10 +22,16 @@ import {
   SyncSensorsActionPayload,
   SyncTemperatureLogsActionPayload,
   SyncTemperatureBreachesActionPayload,
-  FailurePayload,
   SyncSuccessPayload,
 } from './types';
-import { PrepareActionReturn } from '~common/types/common';
+import { FailurePayload, PrepareActionReturn } from '~common/types/common';
+
+export const ENDPOINT = {
+  LOGIN: 'login',
+  SENSOR: 'sensor',
+  TEMPERATURE_LOG: 'temperature-log',
+  TEMPERATURE_BREACH: 'temperature=-breach',
+};
```

Comment:
```suggestion
export const ENDPOINT = {
  LOGIN: '/v1/login',
  SENSOR: '/v1/sensor',
  TEMPERATURE_LOG: '/v1/temperature-log',
  TEMPERATURE_BREACH: '/v1/temperature=-breach',
};
```
Make sense to me to include the v1 as part of the url? If we do a v2 we'd need an app update right?
Although I guess the reverse is true, if we do an app upgrade but server isn't configured for v2? but we can worry about that another time I guess?

Reply:
More important is to fix the the `=` in `temperature-breach`

Reply:
thanks! good save with the temperature-breach endpoint!

re the `/v1/` - ah. oops 💡 I've realised that we don't need `/coldchain/v1/` as part of the configured URL at all, since that is the server endpoint which this app is calling 🙄 that makes the configuration task so much easier! 🙏